### PR TITLE
Add ignore keyword to `FeatureSelector` Transformer

### PR DESCRIPTION
**Pull request by OkonSamuel**
This PR 
1. Adds `ignore` keyword to `FeatureSelector` Transformer similar to `ignore` keyword used in `Standardizer`.
2. allow `features` keyword to accept a function similar to `features` keyword used in `Standardizer`

### CODE REVIEW

**Comment by OkonSamuel** [comment]
might want to add NaiveBayes test back before merging

**Comment by ablaom** [comment]
Looks great. Any chance of adding a test?

File: src/builtins/Transformers.jl [hunk 1]
```diff
@@ -288,12 +289,35 @@ julia> transform(fit!(machine(select2, X)), X)
...
+        ;
+        features::Union{AbstractVector{Symbol}, Function}=Symbol[],
+        ignore::Bool=false
+    )
+        transformer = new(features, ignore)
+        message = MLJBase.clean!(transformer)
+        isempty(message) || throw(ArgumentError(message))
+        return transformer
+    end
+end
```
Comment (ablaom) [inline src/builtins/Transformers.jl:309]: According to the [API docs](https://alan-turing-institute.github.io/MLJ.jl/dev/adding_models_for_general_use/#New-model-type-declarations-and-optional-clean!-method-1) you should not define internal constructors for MLJ models. Same for your changes to Standardizer below (which should not really be in this PR). 

Okay, this is new. Is there a reason you are making keyword constructors *internal* constructors? This is not in the API

**Comment by ablaom** [comment]
@OkonSamuel Assuming this passes CI, you ready for merge?

**Comment by OkonSamuel** [comment]
> @OkonSamuel Assuming this passes CI, you ready for merge?

Yeah

**Pull request merged**